### DISCUSSION TITLE
Snowfall amounts missing from the /gridpoints response

### DISCUSSION THREAD

**Discussion by chrissnell** [weather-gov/api#701] [Q&A] [answered]
I’m making a request for a forecast for a grid area around my location.  This is the URL:

https://api.weather.gov/gridpoints/SLC/100,162/forecast

I’m looking for predicted snowfall amounts.  I see it within `detailedForecast` but I don’t see it anywhere else.  I’d rather not have to parse out snowfall predictions from a block of text.  Surely this is available in structured form somewhere?

**Comment by sullynole** [comment, marked as answer]
The raw data has a snowfallAmount field:  https://api.weather.gov/gridpoints/SLC/100,162

**Reply by chrissnell** [reply]
Thank you!
